pmm-app/.eslintrc

```diff
@@ -89,5 +89,6 @@
     "no-unused-expressions": ["off"],
     "class-methods-use-this": ["off"],
     "react/require-default-props": ["off"],
+    "no-nested-ternary": ["off"],
```

Comment:
Disabled this because I think there's no better way of doing it, see `InfoBox.tsx`

Reply:
> Disabled this because I think there's no better way of doing it, see `InfoBox.tsx`

Actually there is always a way to do it, please check out 
Multi-storey ternary is just too diffcult to read and comprehend )